towhee/runtime/node_config.py

```diff
@@ -178,3 +178,161 @@ def from_dict(triton_info: Dict[str, Any]):
         if 'preferred_batch_size' not in triton_info:
             return TritonServerConf()
         return TritonServerConf(triton_info.get('preferred_batch_size'))
+
+
+# pylint: disable=invalid-name
+class AutoConfig:
+    """
+    Auto configuration for node.
+    """
+    def __init__(self):
+        raise EnvironmentError(
+            'AutoConfig is designed to be instantiated, please using the `AutoConfig.LocalCPUConfig()` etc.'
+        )
+
+    @classmethod
+    def LocalCPUConfig(cls):
+        """
+        Auto configuration to run with local CPU.
+
+        Examples:
+            >>> from towhee.dc2 import pipe, AutoConfig
+            >>> p = (pipe.input('a')
+            ...          .flat_map('a', 'b', lambda x: [y for y in x], config=AutoConfig.LocalCPUConfig())
+            ...          .output('b'))
+        """
+        device_info = {
+            'device': -1
+        }
+        return [device_info]
```

Comment:
Return a config object is better than a list or dict

Reply:
Yeah, and add TowheeConfig for mapping.